tools/tdbg/adminClusterCommands.go

```diff
@@ -0,0 +1,112 @@
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package tdbg
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	"go.temporal.io/server/api/adminservice/v1"
+)
+
+// AdminDescribeCluster is used to dump information about the cluster
+func AdminDescribeCluster(c *cli.Context) error {
+	adminClient := cFactory.AdminClient(c)
+
+	ctx, cancel := newContext(c)
+	defer cancel()
+	clusterName := c.String(FlagCluster)
+	response, err := adminClient.DescribeCluster(ctx, &adminservice.DescribeClusterRequest{
+		ClusterName: clusterName,
+	})
+	if err != nil {
+		return fmt.Errorf("Operation DescribeCluster failed.: %s", err)
+	}
+
+	prettyPrintJSONObject(response)
+	return nil
+}
+
+// AdminListClusters is used to fetch information about all clusters
+func AdminListClusters(c *cli.Context) error {
+	adminClient := cFactory.AdminClient(c)
+	var token []byte
+
+	pageSize := c.Int(FlagPageSize)
+	for more := true; more; more = len(token) > 0 {
+		if more && len(token) > 0 {
+			if !showNextPage() {
+				break
+			}
+		}
+		ctx, cancel := newContext(c)
+		response, err := adminClient.ListClusters(ctx, &adminservice.ListClustersRequest{
+			PageSize:      int32(pageSize),
+			NextPageToken: token,
+		})
+		cancel()
+		if err != nil {
+			return fmt.Errorf("Operation ListClusters failed.: %s", err)
+		}
+		token = response.GetNextPageToken()
+		if len(response.GetClusters()) > 0 {
+			prettyPrintJSONObject(response.GetClusters())
+		}
+	}
+	return nil
+}
+
+// AdminAddOrUpdateRemoteCluster is used to add or update remote cluster information
+func AdminAddOrUpdateRemoteCluster(c *cli.Context) error {
+	adminClient := cFactory.AdminClient(c)
+	ctx, cancel := newContext(c)
+	defer cancel()
+
+	_, err := adminClient.AddOrUpdateRemoteCluster(ctx, &adminservice.AddOrUpdateRemoteClusterRequest{
+		FrontendAddress:               c.String(FlagFrontendAddress),
+		EnableRemoteClusterConnection: c.Bool(FlagConnectionEnable),
+	})
+	if err != nil {
+		return fmt.Errorf("Operation AddOrUpdateRemoteCluster failed.: %s", err)
+	}
+	return nil
+}
+
+// AdminRemoveRemoteCluster is used to remove remote cluster information from the cluster
+func AdminRemoveRemoteCluster(c *cli.Context) error {
```

Comment:
We need to move these 2 commands to tctl and make the api available as Operator API.

Reply:
@yiminc @yycptt would prefer to clear DB commands separately as in here https://github.com/temporalio/tctl/pull/194/files

Reply:
This is not DB command. This is for remote cluster management. We need them to be available via tctl. Those are not for debug, they are for operation.

Reply:
meant to comment here https://github.com/temporalio/temporal/pull/2932#discussion_r887244898

Reply:
added tasks to our board. Resolving this comment. The commands stay for now until we have Operator API and the commands are available in tctl